module/zfs/spa.c

```diff
@@ -4235,6 +4420,8 @@ spa_tryimport(nvlist_t *tryconfig)
 		    poolname) == 0);
 		VERIFY(nvlist_add_uint64(config, ZPOOL_CONFIG_POOL_STATE,
 		    state) == 0);
+		VERIFY(nvlist_add_uint64(config, ZPOOL_CONFIG_IMPORT_TXG,
+		    spa->spa_uberblock.ub_txg) == 0);
```

Comment:
Unfortunately the config nvlist that these keys go in, is generated in `spa_tryimport()` after spa_load() has returned, so there is no such config nvlist to add to within `spa_activity_check()`.  I'll keep it in the same place, and make it conditional on the error value returned.